File: JAVA/Program/Produk.java
import java.util.ArrayList;
public class Produk {
    private int id_produk;
    private String merk;
    private int harga;
    private int stok;

    public Produk(){
        
    }
    // constructor
    public Produk(int id_produk, String merk, int harga, int stok) {
        this.id_produk = id_produk;
        this.merk = merk;
        this.harga = harga;
        this.stok = stok;
    }

    // getter dan setter
    public int getIdProduk() {
        return id_produk;
    }

    public String getMerk() {
        return merk;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public int getHarga() {
        return harga;
    }

    public void setHarga(int harga) {
        this.harga = harga;
    }

    public int getStok() {
        return stok;
    }

    public void setStok(int stok) {
        this.stok = stok;
    }
}

File: PHP/Program/Produk.php
<?php

class Produk{
    private int $id_produk;
    private string $merk;
    private int $harga;
    private int $stok;
    private string $gambar;

    public function __construct(int $id_produk, string $merk, int $harga, int $stok, string $gambar){
        $this->id_produk = $id_produk;
        $this->merk = $merk;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->gambar = $gambar;
    }

    //getter idlaptop
    public function getid_produk(): int {
        return $this->id_produk;
    }

    //setter id_produk
    public function setid_produk(string $id_produk): void {
        $this->id_produk = $id_produk;
    }
    //getter merk
     public function getMerk(): string {
        return $this->merk;
    }
     //setter merk
    public function setMerk(string $merk): void {
        $this->merk = $merk;
    }
    //getter harga
    public function getHarga(): int {
        return $this->harga;
    }
     //setter harga
    public function setHarga(int $harga): void {
        $this->harga = $harga;
    }
    //getter stok
    public function getStok(): int {
        return $this->stok;
    }
     //setter stok
    public function setStok(int $stok): void {
        $this->stok = $stok;
    }
    //getter gambar
    public function getGambar(): string {
    return $this->gambar;
    }      
     //setter gambar
    public function setGambar(string $gambar): void {
    $this->gambar = $gambar;
    }
}

